common/generated_libs/network_protocol/network_protocol.pb.c

```diff
@@ -19,3 +19,8 @@ PB_BIND(SensorReadings, SensorReadings, AUTO)
 
 
 PB_BIND(Alarm, Alarm, AUTO)
+
+
+
+
+
```

Comment:
I don't really care about extra white space, but was that added intentionally?

Reply:
My philosophy is to avoid modifying autogenerated code, so that there aren't spurious diffs if someone regenerates the file.